tests/Tests/Scrapers/BullereiTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace JK\Mittagstisch\Tests;
+
+use JK\Mittagstisch\Scrapers\Bullerei;
+
+class BullereiTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var Bullerei */
+    private $instance;
+
+    public function setUp()
+    {
+        $this->instance = new Bullerei();
+    }
+
+    public function testInstance()
+    {
+        static::assertInstanceOf('JK\Mittagstisch\Scrapers\Bullerei', $this->instance);
+    }
+
+    public function testScrape()
+    {
+        $menus = $this->invokeScrape($this->instance, 'scrape');
+        static::assertInternalType('array', $menus);
+        static::assertContainsOnlyInstancesOf('JK\Mittagstisch\MenuItem', $menus);
```

Comment:
You are right. 
Propably there is some kind of method-splitting necessary to test the regex-matching. I will have look at this "network-stubbing", I don't have any experience about that.
It would be really useful to check or parse against given HTML from the past, so that we can catch several "kinds of postings", e.g. extra <p>-tags, changed string-formatting.